cachi2/core/package_managers/bundler/parser.py

```diff
@@ -8,8 +8,9 @@
 
 import pydantic
 from git import Repo
+from typing_extensions import Self
```

Comment:
**Commit message**:
I'd argue that the main motivation here is to run the validation as part of the model rather than externally and so adding the attribute is just the necessary side effect to get us there. In context of the changes themselves it reads a bit weird, so please reword it accordingly.